src/transform/FooterMenu.css

```diff
@@ -146,3 +146,7 @@ body[dir='rtl'] .pagelib_footer_menu_item_subtitle {
 .pagelib_theme_sepia .pagelib_footer_menu_icon_coordinate {
   background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAFAAAABQCAMAAAC5zwKfAAAABGdBTUEAALGPC/xhBQAAAAFzUkdCAK7OHOkAAAAgY0hSTQAAeiYAAICEAAD6AAAAgOgAAHUwAADqYAAAOpgAABdwnLpRPAAAAbZQTFRFAAAA////gICAqqqqgICAmZlmgICAgICAgIBmdHR0gGpqdnZibW1td3dmeGlpcXFjeWtrbW1hdGhodWpqdmxicWhobW1kcmphb29mc2tjcGhobGxkbW1mcGpkcWxgbWdibWhjcWxib2pgbmphbGhjb2pib2tjbGhhbmtjb2hhbmpjbGlibWpgbGljbmhibWdhbWdhb2lgbWhibGlhbmhjbWdibmlhbmlhbWhgbmlhbmlibmlibWhhbmlibWhhbmlgbWlgbmhhbWlhbGhibWdhbGhgbWlhbmhgbGhhbGlhbmhgbmhhbGlhbWhhbWdibmhhbGhhbGdgbWhhbWhhbWhhbGdhbWhgbWhhbGhgbWhhbGhgbWdhbWdgbWhhbGhhbWhhbGhhbWhhbWdhbGhgbWhhbWdhbWhhbGdhbWhgbGdhbWhhbWhgbWhhbGdgbWhhbWhgbGdhbWhhbGdgbWhhbGhhbGhhbWdgbWhgbGhhbGhhbWdhbGhgbGhhbGhgbWdhbGhgbWhhbWdhbGhgbGhgbWdgbWdhbGhhbWdgbGdhbWhgbGdgbWhhbWdgbGdhbGdgbWhhbGdgbGdgNwzQAQAAAJF0Uk5TAAECAwQFBggKCwwNDg8REhMVFhgaGxwdHh8gISMpLS8xNDU6Ozw+QkNHSElNUFFUWVpbXF1eX2FiZGZobG1ucnd7fH1+f4GChIaHiYuMjY6QkpOWmJmaqaqrrK2vsLW2uLu+v8DDxcbHyMrMzs/Q0dLT1NXW29zf4OLj5OXn6Onq6+zu7/Lz9PX29/j5+vz9/r3wcdoAAAABYktHRAH/Ai3eAAADJElEQVRYw+1YW1sSQRgeBA+QKWDiAQSLjChLOltUCJlSoUkHUA6ZJphkARVhWliBmK4J7z/uYldEAZnd5XK/m/1m3ud5n5n9zkOIJJJIIokw6bZ7A9F0Ohrw2rvFs/U4Y0UAYBgAKMacPaLoOjwMkHRbjWpC1EarOwkwng7BdDJHFjmXoXzL4Moh65AJ41OGwEx3Ht/tnGYQUgrh64sjYagGGBKI9wngyyCsqg6pwsjwZlTFMVPzV8lmEFfxtEcI4TI+mcZk0pSvwwjxs4wDidIRmm2zeQDYmrU1ly6QgIOX/+WYkj1GUkA24vdHskBqpGQZJsvHHz2YOjieDwhaFIQQorAEAd/BIafg4RFvTI7zv+YFpM2HgHkNCxxjZ46hj0InXJzmw4qmHNGswMepLjipCWPg/uAI0pqjkCYN7j8aEKPOV8Ukd+EUzMdBM1LcpZNF2mxmh5tVriFYiQZhYxU37JSEXlhZZQ6WStSCWVaxwktJGICRjYd8VlGJKrJbbJAYEaAkXIaaEEKIFpFqcARaQgghaixTEqYZ9muCvxrsh4lVmDQl4foOHeHOOiXhJ7SefGXWN1sRpyRchI7GKDosUhL6cInGbS6XorCejGKCVWwnOvYERikJB/COJvSWMECb/78Vemsmh7WD5NBbTPFIX+P109cjHulLt/9TVS/BqjL7PNocHybrlYBJvOZRU7ryO/0nFyn933wXn7I3hkhTWRndOl5GmyIY41WX5VE8Lre71mTSllf2cUTl/FoHfa5wsTZqKeT0fJubK9isWTLOZHCVf/s1hfct1ZGWZUwL6A/li3hVHXmJJYWQjlPzHfeq7d/FhlZYT3x2999Q5a55b/ec0K79OjYrwkuXwS3hY4UHq21Hd9o+4LmIOaVpHsEjnapsDm/lIghJ+xc8KV+78fW0uNFM/xs3D1c38Mcgdtgb2mPOlwzM7F0QPz7exi8ulfVv4k4jxtun+NxOCCGnknjWkHlZFsK8nBD5G4SbGjOBK1fxghAvPipJg6R7Aw8f4oeuca8Eg9uFwvZgI98dhguF4ca+ZDy4L73mSCKJJGLkPzZL2BX5qI1YAAAAAElFTkSuQmCC');
 }
+
+.pagelib_theme_sepia .pagelib_footer_menu_icon_reference_list {
+  background-image: url('');
```

Comment:
@berndsi and @joewalsh this one I don't know how to do it, could you please help me?

Reply:
Good question. Maybe @montehurd knows? Also: where do we get the icon for references from?

Reply:
@thesocialdev it can remain empty for now or you can just copy one of the existing ones as a placeholder. The icons will all be updated after https://phabricator.wikimedia.org/T226094 is completed.